homeassistant/components/ecodevices/translations/en.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Device is already configured"
+    },
+    "error": {
+      "cannot_connect": "Failed to connect",
+      "invalid_auth": "Invalid authentication",
+      "unknown": "Unexpected error"
+    },
+    "step": {
+      "user": {
+        "title": "Parameters of Eco-Device",
+        "description": "Set connexion settings and select which input you use.",
+        "data": {
+          "name": "Name",
+          "host": "Host",
+          "port": "Port",
+          "scan_interval": "Seconds between updates",
+          "username": "Username",
+          "password": "Password",
+          "t1_enabled": "Teleinfo 1",
+          "t2_enabled": "Teleinfo 2",
+          "c1_enabled": "Meter 1",
+          "c2_enabled": "Meter 2"
+        }
+      },
+      "params": {
+        "title": "Parameters of Eco-Device entities",
+        "data": {
+          "t1_name": "Teleinfo 1 Name",
+          "t1_unit_of_measurement": "Teleinfo 1 Unit of measurement",
```

Comment:
I am not familiar enough with this device, but is the unit of measurement not standardized? Or is this to support edge cases? They should be able to override this as well via `customize`.

Reply:
Yes it depends on what you connect on the inputs. As it's "standard" to have to customize the input, isn't it better to  customize at the installation than go to advanced `customize` section ?
For the Name, it's easily customizable, but not for device class or unit